lightbox.js

```diff
@@ -146,6 +173,14 @@ function bind() {
   });
 }
 
+function _setCloseIconColor() {
```

Comment:
lol, dang, it worked because it's a global... but should pass that in... good catch